### PR TITLE
initial kivy skeleton

### PR DESCRIPTION
**Pull request by gables**
Initial kivy app that launches a blank canvas.  This is only intended as a starting point.

### CODE REVIEW

File: wandering-warriors/main.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from kivy.app import App
+from kivy.uix.widget import Widget
+from kivy.lang import Builder
+
+
+class Abacus(Widget):
+    pass
+
+
+class DrawingInput(Widget):
```
Comment (DanielBeer) [inline wandering-warriors/main.py]: I'm not going to be picky about naming conventions once we get deeper into the code, but I think it's worth putting "cuneiform" and/or "ledger" in the name here as "drawing input" doesn't really give off the right connotation. My recommendations would be any of "CuneiformDrawingInput", "CuneiformInput", "LedgerDrawingInput", "LedgerInput"

File: wandering-warriors/main.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class Ledger(Widget):
+    pass
+
+
+class AbacusApp(App):
+    def build(self):
+        Builder.load_file("abacus.kv")
+        app = Abacus()
```
Comment (DanielBeer) [inline wandering-warriors/main.py]: I don't think the Abacus widget should be the root of the app. It should probably be a GridLayout which contains the Abacus, Ledger, and ledger input widgets. Their positioning doesn't really matter until we finish the storyboarding

File: wandering-warriors/abacus.kv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<Abacus>:
```
Comment (DanielBeer) [inline wandering-warriors/abacus.kv]: I'm not sure what this file is for and why it just contains an Abacus widget, and why the Abacus widget has a canvas element

**Review by DanielBeer — CHANGES_REQUESTED** [review]

File: wandering-warriors/main.py (hunk 1 above)
Comment (gables) [reply wandering-warriors/main.py]: My reasoning is that at a functional level, the space for taking drawing input is general, could be multi purpose.  It's taking user drawn input, converting it to an analyzable data structure, and passing it off.  Could be cuneiform, or something else.  Thats for the classifier to decide.

File: wandering-warriors/abacus.kv (hunk 3 above)
Comment (gables) [reply wandering-warriors/abacus.kv]: kv file for detailing widgets. https://kivy.org/doc/stable/guide/lang.html .
Just built in the syntax for loading a separate file so we don't have to keep massive kv strings in line with the python code

Comment (gables) [reply wandering-warriors/abacus.kv]: canvas is just a blank thing to load. agian, just a starting place.

File: wandering-warriors/main.py (hunk 2 above)
Comment (gables) [reply wandering-warriors/main.py]: You're right, there will need to be some kind of master object to contain everything.  Not sure exactly what that is yet.  Could be GridLayout, but wasn't ready to assert that.

File: wandering-warriors/main.py (hunk 1 above)
Comment (DanielBeer) [reply wandering-warriors/main.py]: We're not working in a general sense though. This is for the context of this project. This is just one of the three main widgets, and it is the cuneiform/ledger input

**Review by DanielBeer — APPROVED** [review]

**Review by n0remac — APPROVED** [review]

**Pull request merged**